src/option_parser.c

```diff
@@ -187,20 +187,55 @@ int ini_get_bool(char *section, char *key, int def)
         }
 }
 
-char *clean_value(char *value)
+char *clean_value(char *value, int line_num)
 {
-        char *s;
-
-        if (value[0] == '"')
-                s = g_strdup(value + 1);
-        else
-                s = g_strdup(value);
-
-        if (s[strlen(s) - 1] == '"')
-                s[strlen(s) - 1] = '\0';
-
-        return s;
+        char *unparsed = value;
```

Comment:
`unparsed` is overwritten on the next line, there is no reason for the assignment.